core/message-tags-3.3.md

```diff
@@ -80,6 +88,36 @@ Individual tag keys MUST only be used a maximum of once per message. Clients
 receiving messages with more than one occurrence of a tag key SHOULD discard all
 but the final occurrence.
 
+### The `TAGMSG` tag-only event
+
+A new event `TAGMSG` is defined for sending messages with tags but no text content.
+This event MUST be delivered to targets in the same way as `PRIVMSG` and `NOTICE`
+events, taking into account channel membership and modes.
```

Comment:
Yeah can definitely add some more examples, just wondering where to draw the line. Should all possible target types be included as in the PRIVMSG examples from 1459?